doc/host_config_schema/cchost_config.json

```diff
@@ -401,6 +401,23 @@
       "type": ["string", "null"],
       "description": "Path to file (JSON) containing initial node data. It is intended to store correlation IDs describing the node's deployment, such as a VM name or Pod identifier"
     },
+    "attestation": {
+      "type": "object",
+      "properties": {
+        "snp_endorsements_endpoint_type": {
+          "type": "string",
+          "enum": ["Azure", "AMD"],
+          "default": "Azure",
+          "description": "Type of endpoint used to retrieve attestation report endorsement certificates (AMD SEV-SNP only)"
+        },
+        "snp_endorsements_endpoint": {
```

Comment:
I've opted for the less-configurable but simpler option from the operator's perspective, since the URI/parameters construction from the fields in the report are well-known. However, the host may change (e.g. Azure has different test/production endpoints).